scripts/linux/publishReleasePackages.sh

```diff
@@ -253,8 +253,7 @@ publish_to_github()
     branch_name=${BRANCH_NAME/"refs/heads/"/""}
     echo "Branch Name is $branch_name"
 
-    # Note: $(pwd) is the $(Build.SourcesDirectory)
-    source ./iotedge/scripts/linux/github/updateLatestVersion.sh
+    source "$(dirname "$(realpath "$0")")/github/updateLatestVersion.sh"
```

Comment:
What does this $0?

Reply:
$0 is the bash input argument for the current running program.
Basically, get the path of the running program, get the directory, and append it with `/github/*` to formulate a relative path